lib/common/actions/index.js

```diff
@@ -98,6 +96,10 @@ function getErrorMessageFromJson (
   if (json) {
     detail = json.detail
     errorMessage = json.message || JSON.stringify(json)
+    // if status >= 500 and detail is being overrode, add original network error in small text
+    if (status >= 500) {
+      detail = detail ? detail += `\nNetwork error!\n\n(${method} request on ${url})` : `\nNetwork error!\n\n(${method} request on ${url})`
```

Comment:
Just a small nit here: you generally don't want to have an assignment statement like `detail += '\nNetwork ...' follow a ternary that itself is part of an assignment statement. That sentence is a little mind-bendy, but here's how I might fix:
```suggestion
      const networkMessage = `\nNetwork error!\n\n(${method} request on ${url})`
      detail = detail ? detail + networkMessage : networkMessage
```

Reply:
Addressed in https://github.com/ibi-group/datatools-ui/pull/694/commits/a0edbbf579f895d943bad26c7516559d4abea535